features/utils.py

```diff
@@ -122,15 +123,25 @@ def go_to_home_page(browser):
     home_page.flaw_list_exist()
 
 
-def go_to_specific_flaw_detail_page(browser, cve_id):
+def go_to_specific_flaw_detail_page(browser):
     """
     Go to a specific flaw detail page
     """
-    go_to_home_page(browser)
-    home_page = HomePage(browser)
-    home_page.quickSearchBox.clear_text()
-    home_page.quickSearchBox.set_text(cve_id)
-    home_page.click_btn('quickSearchBtn')
+    cve_id = os.getenv("FLAW_ID")
```

Comment:
How to get the FLAW_ID? export the FlAW_ID after finishing the flaw_create_*.feature?

Reply:
Check https://github.com/RedHatProductSecurity/osim/blob/OSIDB-2692/features/steps/flaw_create.py#L135 and https://github.com/RedHatProductSecurity/osim/blob/OSIDB-2692/features/steps/flaw_create.py#L85 .
If you want run a single case locally, just export `FlAW_ID` you want operate.

Reply:
I think if the create flaw function has something wrong, it will block all the following testing, right?

Reply:
Yes. It will.